build.gradle

```diff
@@ -38,7 +38,14 @@ dependencies {
 
 detekt {
     config = files("detekt-config.yml")
-    input = files("$projectDir")
+    input = files(
```

Comment:
why? I think it's not needed, can you review it? Maybe you can try upgrading the detekt version

Reply:
if it doesn't work we can proceed with this way